examples/jacobi/CMakeLists.txt

```
@@ -16,29 +17,37 @@ set(REAL_SIZE_IN_BYTES "8" CACHE STRING "size of the floating point data type")
 set_property(CACHE REAL_SIZE_IN_BYTES PROPERTY STRINGS "8" "4")
 
 set(DEVICE_BACKEND "CUDA" CACHE STRING "type of an interface")
-set_property(CACHE DEVICE_BACKEND PROPERTY STRINGS "CUDA" "HIP")
+set_property(CACHE DEVICE_BACKEND PROPERTY STRINGS "CUDA" "HIP" "ONEAPI" "HIPSYCL")
 
 set(SM "sm_60" CACHE STRING "model of Streaming Multiprocessor")
 set(SM_OPTIONS "sm_20" "sm_30" "sm_50" "sm_60" "sm_70" "gfx906")
 set_property(CACHE SM PROPERTY STRINGS ${SM_OPTIONS})
```

where are options for intel?

---

Add options for intel gpus received with `ocloc compile --help` :
```suggestion
set(SM "sm_60" CACHE STRING "model of Streaming Multiprocessor")
set(SM_OPTIONS "sm_20" "sm_30" "sm_50" "sm_60" "sm_70" "gfx906" "bdw" "skl" "kbl" "cfl" "bxt" "glk" "icllp" "lkf" "ehl" "tgllp" "rkl" "adls" "dg1" "Gen8" "Gen9" "Gen11" "Gen12LP")
set_property(CACHE SM PROPERTY STRINGS ${SM_OPTIONS})
```